segmentation.sh

```diff
@@ -0,0 +1,20 @@
+# Script creates segmentations of files ending  in .png_bw.png.  This file
+# ending is created when the images are run through preprocess.py in Tessie_True
+# -psm is the page segmentation.  Best results were found with --psm 4, --psm 6, --psm 1.
+# Run separate times to create psm 4, psm 6, psm 1
+
+
+lang=storysquadnew
+#set -- **/*.png_bw
+find . -type f -name "*.png_bw.png" -exec sh -c '
+    for img_file do
+        echo -e  "\r\n File: $img_file"
+        mogrify -density 300 "$img_file"
+        OMP_THREAD_LIMIT=1 tesseract --tessdata-dir /mnt/c/Users/dabor/dev/storysquad/Tessie_True/tesseract/tessdata   "${img_file}" "${img_file%.*}"  --dpi 300 --psm 6  --oem 1 -l storysquadset -c page_separator="" hocr
```

Comment:
You are probably gonna have to change the `--tessdata-dir` flag to TESSDATA_PREFIX or something so it looks on their local file structure. This one searches on your specific set up.